modules/openapi-generator/src/main/resources/C-libcurl/apiClient.c.mustache

```diff
@@ -435,8 +453,8 @@ void apiClient_invoke(apiClient_t    *apiClient,
                     curl_easy_setopt(handle, CURLOPT_SSL_VERIFYHOST, 2L);
                 }
             } else {
-                curl_easy_setopt(handle, CURLOPT_SSL_VERIFYPEER, 0L);
-                curl_easy_setopt(handle, CURLOPT_SSL_VERIFYHOST, 0L);
+                curl_easy_setopt(handle, CURLOPT_SSL_VERIFYPEER, 1L);
```

Comment:
`apiClient->sslConfig` should be used to enable and configure TLS.

Reply:
We can discuss this in your new PR. Just putting it here for your consideration.

Reply:
Sure, but isn't it better to default to the safer option? If you disagree, I guess we can just keep applying this patch internally.